src/main/java/spoon/reflect/meta/impl/ModelRoleHandlers.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
This destroys the license check in the CI btw

Reply:
Done, probably a bug of sniper.